charts/pathfinder-rpc-server/values.yaml

```
@@ -0,0 +1,36 @@
+# LetsEncrypt certificates
+acmeEmail:                 circlesubi@gmail.com
+acmeServer:                https://acme-v02.api.letsencrypt.org/directory
+
+# Smart Contracts
+contracts:
+  hubAddress:
+
+
+
+
+# Pathfinder proxy
+pathfinderProxy:
+  port: 8675
+  upstreamServiceEndpoints: http://pathfinder2:54389
+  upstreamHealthEndpoints:  http://pathfinder2-updater:8794
+
+# Pathfinder updater
+pathfinderUpdater:
+  indexerDBConnectionString:
+  indexerWSurl:              ws://blockchain-indexer:8675
+  internalCapacityGraph:     /var/pathfinder2/data/capacity_graph.db
+  externalCapacityGraph:     /var/pathfinder2/data/capacity_graph.db
+  rpcURL:                    http://pathfinder2:54389
+
+postgresql:
+  enabled: true 
+  auth:
+    username:                 doadmin
```

This is a legacy from DigitalOcean and could probably default to something more generic.

```suggestion
    username:                 indexer
```

---

actually no, because the when we do pg_restore it looks for `doamin` role, otherwise postgres throws an error `error: could not execute query: ERROR:  role "doadmin" does not exist`

---

We should eventually leave this free to choose for actual deployments of this chart and set no default, or only specify `doadmin` as an overlay in the actual deployment's values.

The sources of the Chart here will be packed into a TGZ file distributed globally via the GitHub Pages CDN, why we might want to be as generic as possible.

---

yes, you are right. Also part of the documentation